demo_nodes_cpp/src/parameters/parameter_node.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
I know this is based on another demo, but I think the copyright should be updated to 2017 since this is a copy not a move.